spk/salt-minion/src/conf/resource

```diff
@@ -0,0 +1,5 @@
+{
+	"usr-local-linker":{
+		"bin": ["bin/salt-minion"]
+	}
+}
```

Comment:
I would prefere SPK_COMMANDS over dedicated resource file. 
So the the resource file creation is under control of the spksrc framework.

Reply:
I had a hard time trying to figure out how to make a DSM 7 service package. Would be nice with a better wiki follow-this-example :)